frontend/packages/topology/integration-tests/features/topology/topology-editing-app-node.feature

```diff
@@ -50,3 +50,158 @@ Feature: Editing an application
+
+        Examples:
+                  | resource         |
+                  | Deployment       |
+                  | DeploymentConfig |
+
+
+        @regression @to-do
+        Scenario Outline: Advanced image options in Edit deployment/deployment config form
+            Given user has "<resource>" workload "workload-d"
+              And user is on Edit "<resource>" page
+             When user clicks on "Show advanced image options" option
+              And user clicks on Create new secret
+              And user enters Secret name as "new-secret1"
+              And user selects Authentication type as "Image registry credentiials"
+              And user enters Image registry server address as "https://quay.io/repository/kubernetes-ingress-controller/nginx-ingress-controller?tag=latest&tab=tags"
+              And user enters Username and Password as "test1" and "test" respectively
+              And user enters Email as "test1@redhat.com"
+              And user clicks on Create button
+             Then user will see "new-secret1" in secret name dropdown under Pull secret
+
+        Examples:
+                  | resource         |
+                  | Deployment       |
+                  | DeploymentConfig |
+
+
+        @regression @manual
+        Scenario Outline: Additional advanced sections in Edit deployment/deployment config form
+            Given user has "<resource>" workload "workload-d"
+              And user is on Edit "<resource>" page
+             When user clicks on "Pause rollouts" in advanced options
+              And user clicks on "Scaling" option
+             Then user will see "Pause rollouts for this <value>" checkbox under Pause rollouts section
+              And user will see "Replicas" in Scaling section
+
+        Examples:
+                  | resource         | value             |
+                  | Deployment       | deployment        |
+                  | DeploymentConfig | deployment config |
+
+
+        @regression @todo
+        Scenario Outline: Editing deployment through form view
+            Given user has created deployment workload "workload-ad"
+              And user is at the Topology page
+             When user right clicks on the node to open context menu
+              And user selects option "Edit Deployment" from context menu
+              And user selects "RollingUpdate" Strategy type under Deployment Strategy
+              And user enters value of "Maximum number of unavailable Pods" and "Maximum number of surge Pods" as "25%"
+              And user selects value of project, image stream and tag section under images as "Openshift", "" and "latest" respectively
+              And user clicks on "Show advanced container options"
+              And user enters "NAME" as "<env_name>" and "VALUE" as "<env_value>"
+              And user clicks on "Show advanced image options"
+              And user creates a secret "new-secret1" and select it from dropdown in Pull secret
```

Comment:
Are you creating secret in this step. If yes please divide this step into 2 steps

And user creates a secret "new-secret1"
And user selects "new-secret-1" from pull secret dropdown